src/utils/enum/MediumEnum.ts

```diff
@@ -1,13 +1,13 @@
 import D from 'i18n';
 
 export const mediumEnum = {
```

Comment:
could it expose valid entries for mediumEnum as type ? (cf. comment in `getContactAttemptByConfiguration `)